allow >1 store in local and remote clusters

tbg opened the pull request:
and defaults to two stores.

Fixes #3471.

<!-- Reviewable:start -->

[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/3582)

<!-- Reviewable:end -->

mberhault commented on acceptance/cluster/localcluster.go:
```diff
@@ -97,25 +97,25 @@ type Event struct {
+	mu                  sync.Mutex // Protects the fields below
+	dns                 *Container
+	vols                *Container
+	numLocal, numStores int
```
nit: I think we generally give each struct field its own line.

tschottdorf replied:
done.

mberhault commented:
LGTM

tschottdorf commented:
something borked here (tries to bootstrap the wrong thing on docker); I'm looking.

tbg committed:
allow >1 store in local and remote clusters

Wanted to use two Stores by default, but this
crashes. Will investigate separately.

Fixes #3471.

tschottdorf commented:
Set the default to one store. Will figure out separately what breaks when there are more.
